### PR TITLE
first draft for review

### PR DESCRIPTION
**Pull request by aabayomi**

### CODE REVIEW

File: _summaries/lang2018pointpillars_2.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
 TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+
+# Core Idea
+
+The authors proposes as much faster approach by which encoder that learns point cloud representations by stacking learned features as vertical tensors.
```
Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:18]: typo as -> a

Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:18]: It will be good to break this line and expand further on each part. Like what is the problem, the paper is trying to solve (3D object detection perfomance), how (using conv 2D layers and using pillars). You can explain a bit the pillar as well as this is kind of novel. What is the novelty in the paper(novel point cloud encoder and network).

Comment (DartingMelody) [reply _summaries/lang2018pointpillars_2.md:18]: You can also add that PointPillars can work with any standard 2D convolutional detection architecture.

File: _summaries/lang2018pointpillars_2.md [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
+
+* Representation of the 3-D lidar point cloud inputs as pseudo-images
+
+* 2-D convolutional backbone processes the images
+
+* Single-shot detectors regresses to 3D boxes
+
+# Loss function 
+
+The loss formulation combines the sotfmax classification loss, focal loss on object classification  and localization loss.This is optimized using Adam of learning rate of $2 * 10^−4$ with a decay rate 0.8 at every 15 epoch.
```
Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:32]: Typo -> space between . and The

File: _summaries/lang2018pointpillars_2.md [hunk 3]
```diff
@@ -1,19 +1,51 @@
...
+
+![](lang2018pointpillars_2a.png)
+
+* Representation of the 3-D lidar point cloud inputs as pseudo-images
+
+* 2-D convolutional backbone processes the images
+
+* Single-shot detectors regresses to 3D boxes
+
+# Loss function
```
Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:30]: I like that you have discussed the loss function but it will be really good if you can elaborate on how the paper idea is realized technically like the three stages -> encoder conversion of point cloud to sparse pseudo images. 2D convolutional backbone network that takes the pseudo image and learns a high-level representation. And also on detection heads predicting boxes.

File: _summaries/lang2018pointpillars_2.md [hunk 4]
```diff
@@ -1,19 +1,51 @@
...
+# Loss function 
+
+The loss formulation combines the sotfmax classification loss, focal loss on object classification  and localization loss.This is optimized using Adam of learning rate of $2 * 10^−4$ with a decay rate 0.8 at every 15 epoch.
+
+![](lang2018pointpillars_2b.png)
+
+
+# Performance
+
+On KITTI datasets PointPillars  outperforms fusion based methods on cars and cyclists my respect to mean average precision (mAP) compared to lidar-only methods.
```
Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:39]: typo my -> by

Comment (DartingMelody) [reply _summaries/lang2018pointpillars_2.md:39]: You can add here that it performs better than SOTA both in terms of accuracy and speed. Maybe a line summarizing the ablation experiments will also be helpful.

File: _summaries/lang2018pointpillars_2.md [hunk 5]
```diff
@@ -1,19 +1,51 @@
...
+![](lang2018pointpillars_2d.png)
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+
+* PointPillars encoder is computationally efficient and performant on object detection. 
+
+* Data Augmentation is required for good performance
```
Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:51]: I like that you have added a point on data augmentation but it should be a part of the technical implementation section. You can move it there.

File: _summaries/lang2018pointpillars_2.md [hunk 6]
```diff
@@ -1,19 +1,51 @@
...
+PointPillar is faster compared to SOTA on BEV
+
+![](lang2018pointpillars_2d.png)
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+
+* PointPillars encoder is computationally efficient and performant on object detection.
```
Comment (DartingMelody) [inline _summaries/lang2018pointpillars_2.md:49]: You can add a point on lidar point clouds. One point on encoder of how it learns point cloud in a vertical column (pillars) which results in 2D conv layers and hence faster computation.

Comment (DartingMelody) [reply _summaries/lang2018pointpillars_2.md:49]: Another TLDR point could be about its performance compared with previous models or it's generalizability that any backbone can be used.

File: _summaries/lang2018pointpillars_2.md (hunk 3 above)
Comment (DartingMelody) [reply _summaries/lang2018pointpillars_2.md:30]: You can also talk a bit about the inference settings like they used NMS.

File: _summaries/lang2018pointpillars_2.md (hunk 4 above)
Comment (DartingMelody) [reply _summaries/lang2018pointpillars_2.md:39]: You can also add further to complete the summary about the interesting variants which were explored in the paper.

**Comment by DartingMelody** [comment]
@aabayomi, I like that your summary captures the basic idea behind the paper. But It will be very good to explain a few things further. I have left comments which may help you to complete your summary. Thanks!

**Comment by aabayomi** [comment]
@DartingMelody Thank you. I have made the changes.

**Review by zhaoyue-zephyrus — APPROVED** [review]
